src/Microsoft.ML.StandardLearners/Standard/Online/doc.xml

```diff
@@ -13,8 +13,8 @@
         and an option to update the weight vector using the average of the vectors seen over time (averaged argument is set to True by default).
       </remarks>
     </member>
-    <example>
-      <example name="OGD">
+    <example name="OGD">
```

Comment:
>example [](start = 5, length = 7)

is this not an error to have nested examples?

Reply:
Yea I see it's used everywhere. I wonder why though?

---
In reply to: [205926076](https://github.com/dotnet/machinelearning/pull/592#discussion_r205926076) [](ancestors = 205926076)

Reply:
it is legal in xml. 
The internal example node is needed by the documentation to display the snippet as an example. 

I needed a named wrapper to get the node, and ...just called it example. 

---
In reply to: [205926101](https://github.com/dotnet/machinelearning/pull/592#discussion_r205926101) [](ancestors = 205926101,205926076)